test/evaluation/evaluationClient.spec.ts

```diff
@@ -14,7 +14,14 @@ describe('evaluate', () => {
 
   it('should return error with null vc in inputDescriptors doesn\'t match', function () {
     const presentationDefinition: PresentationDefinition = getFile('./test/dif_pe_examples/pd/pd-simple-schema-age-predicate.json').presentation_definition;
-    const vpSimple = new VP(new Presentation(null, null, null, null, HOLDER_DID, null));
+    const vpSimple: VerifiablePresentation = {
+      '@context': null,
+      type: null,
+      proof: null,
+      verifiableCredential: null,
+      holder: HOLDER_DID,
+      presentationSubmission: null
+    };
```

Comment:
instantiation should be cleaner. 
Using 'new' perhaps.

Reply:
Nope it is not. You are looking at this with a Java hat on